app/views/timelogs/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="well">
+  <p>Your timelogs:</p>
+  <% @timelogs.each_with_index do |timelog, index| %>
+    <div class="well">
```

Comment:
We had similar situation with users.
http://guides.rubyonrails.org/layouts_and_rendering.html
Read about rendering collections.